Makefile

```diff
@@ -40,6 +38,15 @@ update-google-otel-components update-otelopscol-components: export OTEL_CONTRIB_
 update-google-otel-components update-otelopscol-components: install-tools
 	cd $(COMPONENT_DIR) && PATH="$(TOOLS_DIR):${PATH}" $(MAKE) update-components
 
+.PHONY: test-google-otel-components test-otelopscol-components
+
+test-google-otel-components: COMPONENT_DIR := components/google-built-opentelemetry-collector
+
+test-otelopscol-components: COMPONENT_DIR := components/otelopscol
+
+test-google-otel-components test-otelopscol-components:
+	cd $(COMPONENT_DIR) && $(MAKE) test-components
```

Comment:
Looks like the tests on components are still not running: https://screenshot.googleplex.com/5GpubrWH7DQMvfw

Those `test-` targets here are calling the `target-all-modules` target in the components directory subfolders, and `go.work` is not executed before them.

Reply:
Yep thanks for catching that too. Done in [39480b3](https://github.com/GoogleCloudPlatform/opentelemetry-operations-collector/pull/292/commits/39480b345711d5d773aca09fd5b53faeb56dbd87)